page/about-jquery/how-to-setup-jquery.md

```diff
@@ -0,0 +1,108 @@
+<script>{
+  "title": "How To Setup jQuery",
+  "level": "beginner"
+}</script>
+
+### jQuery: Setup
+
+This is a basic tutorial, designed to help you get started using jQuery. Start by creating the following HTML page and name it `index.html`, and open it in the browser:
+
+```
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Demo</title>
+</head>
+<body>
+  <a href="http://jquery.com/">jQuery</a>
+  <script src="jquery.js"></script>
+  <script src="demo.js"></script>
+</body>
+</html>
+```
+
+The `src` attribute in the first `<script>` element must point to a copy of jQuery. Download a copy of jQuery from the [Downloading jQuery](http://jquery.com/download/) page and store the `jquery.js` file in the same directory as your HTML file.
+
+*Note: When you download jQuery, the file name may contain a version number, e.g., `jquery-x.y.z.js`. Make sure to either rename this file to `jquery.js` or update the `src` attribute of the `<script>` element to match the file name.*
+
+Create another file in the same directory called `demo.js`. The `src` attribute in the second `<script>` element must point to the location of this JavaScript file, which will contain the code that will run on the page.
+
+### Launching Code on Document Ready
+
+To run code as soon as the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction) is loaded and ready to be manipulated, jQuery has a statement known as the [ready event](http://api.jquery.com/ready/):
```

Comment:
This may of been correct to call it an event earlier, but anymore we should probably call this a "ready method"
```js
$( doucment ).ready( function {
  // Your code here.
});

// Or the shorthand:
$(function() {
  // document is ready!
});